prometheus/summary.go

```diff
@@ -158,6 +163,17 @@ func newSummary(desc *Desc, opts SummaryOpts, labelValues ...string) Summary {
 		panic(errInconsistentCardinality)
 	}
 
+	for _, n := range desc.variableLabels {
+		if n == "quantile" {
```

Comment:
I think "quantile" and "le" should be constants in https://github.com/prometheus/client_golang/blob/master/model/labelname.go#L20-L36